libs/librtlnumber/src/include/internal_bits.hpp

```diff
@@ -451,13 +478,30 @@ namespace BitSpace {
             std::string to_return = "";
             for(size_t address=0x0; address < this->size(); address++)
             {
-                to_return.push_back(BitSpace::bit_to_c(this->get_bit(address)));
+                char value = BitSpace::bit_to_c(this->get_bit(address));
+                if(big_endian)
+                {
+                    to_return.push_back(value);
+                }
+                else
+                {
+                    to_return.insert(0,1,value);
+                }
             }
 
-            if(!big_endian)
-                return std::string(to_return.crbegin(), to_return.crend());
-            else
-                return to_return;              
+            return to_return;              
+        }
+
+        std::string to_printable()
```

Comment:
I still don't think printing everything as a string is the way to go. it'd be much cleaner and more interpretive if you print numbers as numbers, and strings as strings. you could just have a string flag, check it, and print either a number or a string. no Odin code bulking this way, and no weird chars for other people to look at and go "what is this gibberish??" when they read it - everyone's happy :-) 

e.g. for value = 01010010: if is_string = true, print R; if is_string = false, print 82 (or 0x52, if you think hex is a better way to go instead of decimal)